app/controllers/dashboard_controller.rb

```diff
@@ -2,6 +2,11 @@ class DashboardController < ApplicationController
   before_action :authenticate
 
   def show
-    
+    main_address = MainAddress.find_by(user_id: current_user.id)
+    if main_address
+      @main_address = main_address
+    else
+      @main_address = MainAddress.new
+    end
   end
```

Comment:
I think we should change this later to note fake an address which might create unexpected bugs later. 
We should probably render the new_main_address form if there is no main address.